carts/models.py

```diff
@@ -84,6 +84,11 @@ def save(self, *args, **kwargs):
             None
         """
         # pylint: disable=no-member
+        self.product.stock_quantity -= self.quantity
```

Comment:
Is the cart item is updated? would it always reduce the stock quantity? e.g. if user adds to the cart it's fine but what if user reduces the quantity.?